From 4ed52a563349248e8832e28c526c95b0c0834702 Mon Sep 17 00:00:00 2001
From: Feng Xiao <xf@rock-chips.com>
Date: Mon, 14 Mar 2016 17:52:04 +0800
Subject: [PATCH] ARM64: dts: rk3366: assigned parents for vop dclks

For sheep board, we have decided to assign vop full for
use with HDMI. And we can also change it in the board
dts in the further.

Change-Id: Id966615c84cef50f0e8d849e3840434ba7f7b7ec
Signed-off-by: Feng Xiao <xf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3366.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3366.dtsi b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
index 9d9c36c68059..06e6ea0004ef 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
@@ -662,17 +662,21 @@
 		#clock-cells = <1>;
 		#reset-cells = <1>;
 		assigned-clocks =
+			<&cru DCLK_VOP_FULL>, <&cru DCLK_VOP_LITE>,
 			<&cru PLL_CPLL>, <&cru PLL_GPLL>,
 			<&cru PLL_NPLL>, <&cru PLL_MPLL>,
 			<&cru PLL_WPLL>, <&cru PLL_BPLL>,
 			<&cru ACLK_VOP_FULL>, <&cru ACLK_VOP_LITE>,
 			<&cru HCLK_VOP_LITE>,<&cru HCLK_VOP_LITE>;
 		assigned-clock-rates =
+			<0>, <0>,
 			<750000000>, <576000000>,
 			<594000000>, <594000000>,
 			<960000000>, <520000000>,
 			<375000000>, <288000000>,
 			<100000000>, <100000000>;
+		assigned-clock-parents =
+			<&cru SCLK_MPLL_SRC>, <&cru PLL_NPLL>;
 	};
 
 	grf: syscon@ff770000 {
-- 
2.35.3

